test_data/NAME/NAMEII_no_time_averaging.txt

```diff
@@ -0,0 +1,30 @@
+NAME III (version 7.2)                                          
```

Comment:
```suggestion
NAME II (version 7.2)                                          
```

?

Reply:
No I don't think that is actually correct.
This is just what confused me in mis-naming the file.
According to @bsherratt, IIUC the top line tells what software version created the file, not the format it is in.
So this is a NAME-II file, created with NAME-III

Reply:
and likewise, the version is the software version not the format version